src/components/organisms/topbar.tsx

```diff
@@ -1,12 +1,16 @@
 import React from "react"
 import Avatar from "../atoms/avatar"
+import Button from "../fundamentals/button"
 import NotificationBell from "../molecules/notification-bell"
 import SearchBar from "../molecules/search-bar"
 
 const Topbar: React.FC = () => {
   return (
     <div className="w-full min-h-topbar max-h-topbar pr-xlarge pl-base bg-grey-0 border-b border-grey-20 sticky top-0 flex items-center justify-between z-50">
       <SearchBar />
+      <Button variant="primary" size="large">
+        Heeey
+      </Button>
```

Comment:
I guess it's for testing purposes, but should we omit it from the PR 😅

Reply:
😅 fixed

Reply:
come on oli .. it's called being polite to users :laughing:

Reply:
Haha 😆